android/framework/encode/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 add_library(gfxrecon_encode STATIC "")
 
+add_library(perfetto STATIC ${CMAKE_CURRENT_LIST_DIR}/../../../../perfetto/sdk/perfetto.cc)
```

Comment:
What does this path translate into.  I needed to change this to:

/home/ianelliott/android/main1/vendor/arm/mali/valhall/cinstr/production/third_party/perfetto/vendor/sdk

where `/home/ianelliott/android/main1` is one of my Android build trees.